src/audio/drc/typedef.h

```diff
@@ -0,0 +1,95 @@
+/*
+ * File: typedef.h
+ *
+ */
+ // SPDX - License - Identifier: BSD - 3 - Clause
+ //
+ //Copyright(c) 2021 Intel Corporation.All rights reserved.
+ //
+ //Author : Shriram Shastry <malladi.sastry@linux.intel.com>
+
+#ifndef TYPEDEF_H
+#define TYPEDEF_H
+
+/* Include Files */
+#include "stdint.h"
+#ifndef __cplusplus
+#include <stdbool.h>
+#endif
+
+#ifdef __cplusplus
+
+extern "C" {
+
+#endif
+
+
+  /*=======================================================================*
+   * Fixed width word size data types:                                     *
+   *   int8_t, int16_t, int32_t     - signed 8, 16, or 32 bit integers     *
+   *   uint8_t, uint16_t, uint32_t  - unsigned 8, 16, or 32 bit integers   *
+   *   real32_t, real64_t           - 32 and 64 bit floating point numbers *
+   *=======================================================================*/
+  typedef signed char int8_t;
```

Comment:
same, these are all defined in public headers, we should not be redefining stuff